There is a nonsense in the configure script which pops up an error message and
potentially have consequence on following command.

https://bugs.launchpad.net/hplip/+bug/1881331

--- hplip-3.25.6/configure.in.orig
+++ hplip-3.25.6/configure.in
@@ -30,7 +30,7 @@
 AC_INIT([HP Linux Imaging and Printing], [3.25.6], [3.25.6], [hplip])
 
 #AM_INIT_AUTOMAKE([1.9 foreign])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([subdir-objects])
 AC_DISABLE_STATIC
 
 # Checks for programs.
@@ -257,7 +257,7 @@
    test `sh ./createPPD.sh -f` == 0
 else
    AC_MSG_RESULT(no)
-   test `sh ./createPPD.sh -q` == 0
+   ./createPPD.sh -q
 fi
 AM_CONDITIONAL(HPLIP_CLASS_DRIVER, test x$class_driver = xyes)
 
